extension.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Semantic Glossary",
-	"version": "5.0.1",
+	"version": "5.0.2",
```

Comment:
I think we should only update this once we have a release date.

Reply:
I would suggest to release it whenever this MR has been merged, since its breaking stuff. Anyways, i reverted the version bump.